.travis.yml

```diff
@@ -37,6 +38,8 @@ jobs:
 
   include:
     - stage: Coding standard
+      install:
+          - travis_retry composer install --no-interaction --prefer-dist --ignore-platform-reqs
```

Comment:
In this case, there would be multiple options, all preferrable to `--ignore-platform-reqs`:
1. when running `composer update` locally, make sure you have the same version installed as travis (from the error message, this seems to be 1.3.2)
2. Alternatively, use a platform config in composer.json to lock `ext-mongodb` to the version on travis, this has other downsides though
3. Install a newer version of `ext-mongodb` on travis using `pecl install mongodb`

Using `--ignore-platform-reqs` can (and likely will at some point in time) cause errors because the new MongoDB library version relies on certain features in the driver which aren't present because it's running an older version than required. My suggestion would be to pick options 1 or 3.